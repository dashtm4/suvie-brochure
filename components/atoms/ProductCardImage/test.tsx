import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ProductCardImage } from '.'

describe('<ProductCardImage />', () => {
  const imageSrc = 'image.jpg'
  const alt = 'Test alt'

  it('should renders correct <ProductCardImage />', () => {
    const wrapper = shallow(<ProductCardImage imageSrc={imageSrc} alt={alt} />)

    expect(wrapper.find('Image').prop('src')).toMatch(imageSrc)
    expect(wrapper.find('Image').prop('alt')).toMatch(alt)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
