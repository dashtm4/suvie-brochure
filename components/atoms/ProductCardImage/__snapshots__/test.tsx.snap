// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductCardImage /> should renders correct <ProductCardImage /> 1`] = `
<figure
  className="relative w-full h-48 mb-3"
  role="none"
>
  <Image
    alt="Test alt"
    layout="fill"
    objectFit="contain"
    src="image.jpg"
  />
</figure>
`;
